dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -102,13 +103,13 @@
 	private Set<String> foldersSet;
 	private Set<String> htmlPagesSet;
 	private Set<String> templatesSet;
-	private Set<String> structuresSet;
+	private Set<String> contentTypesSet;
 	private Set<String> containersSet;
 	private Set<String> fileAssetContainersSet;
 	private Set<String> contentsSet;
 	private Set<String> linksSet;
 	private Set<String> ruleSet;
-	private Set<String> solvedStructures;
+	private Set<String> solvedContentTypes;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'solvedContentTypes' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5458951)